data/json/scores/achievements.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "id": "achievement_kill_zombie",
+    "type": "achievement",
+    "description": "One down, billions to go\u2026",
+    "requirements": [ { "event_statistic": "num_avatar_zombie_kills", "is": ">=", "target": 1 } ]
+  },
+  {
+    "id": "achievement_swim_short_distance",
+    "type": "achievement",
+    "description": "A Swimming Merit Badge",
+    "requirements": [ { "event_statistic": "num_moves_swam", "is": ">=", "target": 10000 } ]
+  },
+  {
+    "id": "achievement_swim_short_distance_underwater",
+    "type": "achievement",
+    "description": "There's Nothing Down Here",
+    "requirements": [ { "event_statistic": "num_moves_swam_underwater", "is": ">=", "target": 1000 } ]
+  },
+  {
+    "id": "achievement_marathon",
+    "type": "achievement",
+    "description": "Pheidippides Was A Hack",
+    "requirements": [ { "event_statistic": "num_moves_ran", "is": ">=", "target": 42196 } ]
```

Comment:
Yup, distance of a marathon +1 meter. I also agree on the name / description differentiation, because I actually toyed with putting that "run a marathon" into the description here directly with the name, but that looked terrible, and I really didn't want to get into reworking the UI part of this (and I'm glad I didn't because I didn't see your achievement PR that updates it until this morning). It'd be good to have the name, narrative description, and then the statistical counters.